theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -418,26 +416,20 @@ def test_pooling3d():
     if not cuda.dnn.dnn_available() or cuda.dnn.version() < (3000, 3000):
         raise SkipTest(cuda.dnn.dnn_available.msg)
 
-    mode_without_gpu_ref = theano.compile.mode.get_mode(
```

Comment:
I've added it back, including the comment you proposed. I originally removed this custom mode because it wasn't in the `test_pooling` function for 2D.